code/modules/station_goals/shield.dm

```diff
@@ -134,6 +134,31 @@
 	speed_process = TRUE
 	var/kill_range = 14
 
+/obj/machinery/satellite/meteor_shield/sci
+	name = "\improper Meteor Shield Satellite"
+	desc = "A station made meteor point-defense satellite."
+	mode = "M-SHIELD"
+
+/obj/item/disk/meteor
+	name = "Meteor Shield Upgrade Disk"
+	desc = "A floppy disk that allows meteor shields to fire at longer ranges and lowers meteor drawing from gravitational fields.."
+
+/obj/machinery/satellite/meteor_shield/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/disk/meteor))
+		to_chat(user, "<span class='notice'>The disk uploads better tracking and rang modification software.</span>")
```

Comment:
focusing, not range modification, or just remove that part

Reply:
This only improves the range by 3 tiles, nothing more. This IC feedback is 100% vaild